Android/app/src/main/res/layout/activity_my_trip_info.xml

```diff
@@ -1,147 +1,159 @@
+        tools:context="io.github.project_travel_mate.destinations.description.FinalCityInfoActivity"
+        >
 
-        <ImageView
-            android:id="@+id/city_image"
+        <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+            xmlns:tools="http://schemas.android.com/tools"
             android:layout_width="match_parent"
-            android:layout_height="200dp"
-            android:contentDescription="@string/empty_description"
-            android:scaleType="centerCrop"
-            android:src="@drawable/placeholder_image" />
-
-        <ProgressBar
-            android:id="@+id/progress_bar"
-            android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:layout_gravity="center"
-            android:visibility="gone"/>
+            android:orientation="vertical"
+            tools:context="io.github.project_travel_mate.destinations.description.FinalCityInfoActivity">
 
-        <TextView
-            android:id="@+id/city_name"
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:fontFamily="sans-serif-bold"
-            android:padding="10dp"
-            android:textColor="@color/black"
-            android:textSize="25sp"
-            android:textStyle="bold" />
+            <ImageView
+                android:id="@+id/city_image"
+                android:layout_width="match_parent"
+                android:layout_height="200dp"
+                android:contentDescription="@string/empty_description"
+                android:scaleType="centerCrop"
+                android:src="@drawable/placeholder_image" />
 
-        <LinearLayout
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:orientation="horizontal">
-
-            <EditText
-                android:id="@+id/trip_name"
-                android:layout_width="307dp"
-                android:layout_height="match_parent"
-                android:background="@android:color/transparent"
-                android:cursorVisible="false"
-                android:ellipsize="end"
-                android:focusableInTouchMode="false"
+            <ProgressBar
+                android:id="@+id/progress_bar"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_gravity="center"
+                android:visibility="gone"/>
+
+            <TextView
+                android:id="@+id/city_name"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
                 android:fontFamily="sans-serif-bold"
                 android:padding="10dp"
                 android:textColor="@color/black"
-                android:textSize="15sp"
-                android:visibility="invisible" />
+                android:textSize="25sp"
```

Comment:
I meant these other changes in the file

Reply:
very strange, I havent edited that. I fix it.